Block/Callback/Ordersummary.php

```diff
@@ -102,7 +111,7 @@ public function __construct(
     /**
      * Get orderif from param
      *
-     * @return id
+     * @return mixed
```

Comment:
Isn't this a string? Maybe a null|string, mixed seams very wide for something like this?

Reply:
Done

Reply:
it still says mixed?